css-view-transitions-1/Overview.bs

```diff
@@ -197,38 +197,59 @@ urlPrefix: https://html.spec.whatwg.org/multipage/rendering.html; type: dfn;
 
 # Pseudo-elements # {#pseudo}
 
-	The following <dfn export>view-transition pseudo-elements</dfn> represent the various items being animated.
+## Pseudo-element parents ## {#parent-pseudo}
 
-	The ''::view-transition'' pseudo-element acts as a grouping element for other [=view-transition pseudo-elements=]
-	and has the document's [=document element=] as its [=originating element=].
+	A <dfn>pseudo-element parent</dfn> is a mixin that has <dfn for="pseudo-element parent">children</dfn>,
+	a [=/list=] of [=pseudo-elements=].
+	Initially an empty [=/list=].
 
-	Note: For example, '':root::view-transition'' selector matches this pseudo-element,
-	but ''div::view-transition'' does not.
+	A [=pseudo-element parent=]'s [=pseudo-element parent/children=] are laid out in the order they appear in the [=/list=], according to their styles.
 
-	Other [=view-transition pseudo-elements=] take a <<pt-name-selector>> argument
-	to specify which elements named with 'view-transition-name' are affected.
+	When a [=pseudo-element=] |pseudo| is part of a [=pseudo-element parent=] |parent|'s [=pseudo-element parent/children=],
+	and |parent| is a [=pseudo-element=],
+	then |pseudo|'s [=originating pseudo-element=] is |parent|.
 
-	There can be multiple pseudo-elements of the same type,
-	one for each 'view-transition-name' participating in a transition.
+	<div algorithm>
+		To get the <dfn for="pseudo-element parent">descendants</dfn> of a [=pseudo-element parent=] |parent|, perform the following steps.
+		They return a [=/list=] of [=pseudo-elements=].
```

Comment:
I don't think you need this line

Reply:
It's the defined format in https://infra.spec.whatwg.org/#algorithm-declaration